classes/04class/exercises/c04-iac01/mrcsmonteiro/iam.tf

```diff
@@ -0,0 +1,49 @@
+resource "aws_iam_role" "ec2_s3_role" {
+  name               = "ec2_s3_role"
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "ec2.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": "EC2Role"
+    }
+  ]
+}
+EOF
```

Comment:
This can help for templating big text files: https://registry.terraform.io/providers/hashicorp/template/latest/docs/data-sources/file